src/transformStateWithClones.js

```diff
@@ -7,7 +7,40 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const initialStateCopy = JSON.parse(JSON.stringify(state));
```

Comment:
why do you need to parse and stringify this object? You can just use `spread` oerator
```suggestion
  const initialStateCopy = {...state};
```